src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -179,20 +186,33 @@ public static void Beep()
     [OuterLoop] // makes noise, not very inner-loop friendly
     public static void BeepWithFrequency()
     {
-        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-        {
-            Assert.Throws<ArgumentOutOfRangeException>(() => Console.Beep(36, 200));
-            Assert.Throws<ArgumentOutOfRangeException>(() => Console.Beep(32768, 200));
-            Assert.Throws<ArgumentOutOfRangeException>(() => Console.Beep(800, 0));
-            Assert.Throws<ArgumentOutOfRangeException>(() => Console.Beep(800, -1));
+        // Nothing to verify; just run the code.
+        Console.Beep(800, 200);
+    }
```

Comment:
Yup - I suggest we run outerloop tests for this before merging to avoid any accidental errors like this one.